src/components/featurescomponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+// Not sure where to pass in feature state data... this is just a place holder
```

Comment:
None of this is state, its just data that can be passed in from props.

If you want, you can store the props in a constants file.